### PR TITLE
css-positioning

### PR DESCRIPTION
**Pull request by HuynhCynthia**
Screen Shot 2023-01-27 at 10 15 13 AM
Screen Shot 2023-01-27 at 10 34 00 AM
Screen Shot 2023-01-27 at 10 34 06 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  /* your code here */
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
```
Comment (JenBowman1) [inline css-positioning/absolute.css:41]: please add position: absolute here!

File: css-positioning/relative.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```
Comment (JenBowman1) [inline css-positioning/relative.css:10]: if you add position: relative here, you don't need to add it to each individual pokemon

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
just a couple small changes!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**